adapters/lora-vertx/src/main/java/org/eclipse/hono/adapter/lora/LoraMetaData.java

```diff
@@ -0,0 +1,243 @@
+/**
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+
+package org.eclipse.hono.adapter.lora;
+
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * A container for meta information contained in Lora
+ * messages.
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public final class LoraMetaData {
```

Comment:
Same here, for every metric there should be a clear and explicit definition of the unit of measurement.